Update README.MD  to add Systemd Couch2pg Config (#1)
nomulex/medic-couch2pg: README.md

end script
```
 - The service is then a standard service, e.g. `service couch2pg-example-client start`
 
 ### Installing as a service using Systemd (18.04.3 LTS [Bionic Beaver])

> To setup couch2pg using systemd  is also pretty simple. You will need to have sudo rights to the server and then follow the steps listed below:

Reviewer: ```suggestion
To setup couch2pg using systemd is also pretty simple. You will need to have sudo rights to the server and then follow the steps listed below:
```

>  - Install git and clone this repo onto your server, check out the relevant tag `git checkout tag_id`, and run `npm install`. 

Reviewer: You should almost never use `npm install` as a general "install all the deps" command these days, as it doesn't respect package locks and so may install newer packages than we'd want it to.

```suggestion
 - Install git and clone this repo onto your server, check out the relevant tag `git checkout tag_id`, and run `npm ci`. 
```

 - Create a systemd unit file for your project `sudo` create `/etc/systemd/system/couch2pg-sample-client.service`

>  - As we are going to put passwords in this file, you want to `chown o-r /etc/systemd/system/couch2pg-sample-client.service` so that only root can read it.

Reviewer: ```suggestion
 - As we are going to put passwords in this file, you want to `sudo chmod o-r /etc/systemd/system/couch2pg-sample-client.service` so that only root can read it.
```

>  - Edit this file and configure the couch2pg system unit. It could be something simillar to this;

Reviewer: ```suggestion
 - Edit this file and configure the couch2pg system unit. It could be something similar to this;
```
